TrackerAlignment/inc/Mille.h

```diff
@@ -0,0 +1,78 @@
+#ifndef MILLE_H
+#define MILLE_H
+
+/** \file
+ *  Define class Mille.
+ *
+ *  \author Gero Flucke, University Hamburg, 2006
+ *
+ *  \copyright
+ *  Copyright (c) 2009 - 2015 Deutsches Elektronen-Synchroton,
+ *  Member of the Helmholtz Association, (DESY), HAMBURG, GERMANY \n\n
+ *  This library is free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU Library General Public License as
+ *  published by the Free Software Foundation; either version 2 of the
+ *  License, or (at your option) any later version. \n\n
+ *  This library is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Library General Public License for more details. \n\n
+ *  You should have received a copy of the GNU Library General Public
+ *  License along with this program (see the file COPYING.LIB for more
+ *  details); if not, write to the Free Software Foundation, Inc.,
+ *  675 Mass Ave, Cambridge, MA 02139, USA.
+ */
```

Comment:
Someone should check that this is all compatible with Offline's current apache license

Reply:
Ah, I'm not sure about this either. Someone else will have to answer. 

If necessary I can write my own class to write the Millepede files. It should be fairly straightforward to do and `Mille` as it is now doesn't support compression or writing doubles. (but the 'pede' software does support reading compressed files of doubles)